dex/testing/polygon/harness.sh

```diff
@@ -235,14 +303,33 @@ EOF
 
 cat > "${NODES_ROOT}/harness-ctl/sendtoaddress" <<EOF
 #!/usr/bin/env bash
-"${HARNESS_DIR}/${NAME}" "attach ${NODE_DIR}/bor.ipc --preload ${NODES_ROOT}/harness-ctl/send.js --exec sendLegacyTx(\"${ALPHA_ADDRESS}\",\"\$1\",\$2*1e18)"
+"${HARNESS_DIR}/alpha" "--preload ${NODES_ROOT}/harness-ctl/send.js --exec sendLegacyTx(\"${ALPHA_ADDRESS}\",\"\$1\",\$2*1e18)"
 EOF
 chmod +x "${NODES_ROOT}/harness-ctl/sendtoaddress"
 
-# Providing a config file overwrites everything provided via cli so we have to
-# provided everything via the config file: See:
-# https://vscode.dev/github/maticnetwork/bor/blob/develop/internal/cli/server/command.go#L77
-tmux send-keys -t "$SESSION:1" "bor server --config ${NODE_DIR}/bor.toml 2>&1 | tee ${NODE_DIR}/${NAME}.log" C-m
+echo "Mining a few more blocks"
+${HARNESS_DIR}/mine-alpha 6
+
+TESTING_ADDRESS="C26880A0AF2EA0c7E8130e6EC47Af756465452E8"
+SEND_AMT=5000000000000000000000
+TEST_TX_HASH=$(${HARNESS_DIR}/alpha --preload "${NODES_ROOT}/harness-ctl/send.js" --exec "sendLegacyTx(\"${ALPHA_ADDRESS}\",\"${TESTING_ADDRESS}\",${SEND_AMT})" | sed 's/"//g')
+echo "ETH transaction to use in tests is ${TEST_TX_HASH}. Saving to ${NODES_ROOT}/test_tx_hash.txt"
+cat > "${NODES_ROOT}/test_tx_hash.txt" <<EOF
+${TEST_TX_HASH}
+EOF
+
+TEST_BLOCK10_HASH=$(${HARNESS_DIR}/alpha --exec "eth.getHeaderByNumber(10).hash" | sed 's/"//g')
+echo "ETH block 10 hash to use in tests is ${TEST_BLOCK10_HASH}. Saving to ${NODES_ROOT}/test_block10_hash.txt"
+cat > "${NODES_ROOT}/test_block10_hash.txt" <<EOF
+${TEST_BLOCK10_HASH}
+EOF
+
+# TEST_TOKEN_CONTRACT_ADDR=$(${HARNESS_DIR}/alpha --preload ${NODES_ROOT}/harness-ctl/contractAddress.js --exec "contractAddress(\"${TEST_TOKEN_CONTRACT_HASH}\")" | sed 's/"//g')
+# echo "Test Token contract address is ${TEST_TOKEN_CONTRACT_ADDR}. Saving to ${NODES_ROOT}/test_token_contract_address.txt"
+# cat > "${NODES_ROOT}/test_token_contract_address.txt" <<EOF
+# ${TEST_TOKEN_CONTRACT_ADDR}
+# EOF
```

Comment:
I think we can remove this if not useful yet.

Reply:
I'm going to follow up quickly with some contracts.

Reply:
Okay.